sdks/java/io/snowflake/src/test/java/org/apache/beam/sdk/io/snowflake/test/unit/write/SnowflakeIOWriteTest.java

```diff
@@ -129,7 +129,7 @@ public void writeToExternalWithMapperTest() throws SnowflakeSQLException {
   }
 
   @Test
-  public void writeToExternalWithKVInput() {
+  public void writeToExternalWithKVInput() throws SnowflakeSQLException {
```

Comment:
Can we remove `External`? Or do you mind removing all `External` in all tests. Since we don't distinguish Internal/External write, it would look better to have shorter test names.